tests/net/net_pkt/src/main.c

```diff
@@ -1178,6 +1178,79 @@ static void test_net_pkt_append_memset(void)
 	net_pkt_unref(pkt);
 }
 
+static struct net_buf *test_net_buf_make(void *data, size_t len)
+{
+	struct net_buf *nbuf = k_calloc(1, sizeof(struct net_buf));
```

Comment:
Note that his might work with test application, but in real app the net_buf cannot be allocated using malloc related functions because proper net_buf initialization would not be done in that case.

Reply:
The unit test as a whole doesn't compile, this is the only way to unit test it now. We will need to have more flexible allocator anyway, let's keep this for now.